src/components/ClipboardAwarePasteButton.tsx

```diff
@@ -31,13 +34,35 @@ export default function ClipboardAwarePasteButton({
   }
 
   return (
-    <Button
-      text={t('paste')}
-      type={BtnTypes.ONBOARDING_SECONDARY}
-      rounded={false}
-      size={BtnSizes.FULL}
-      onPress={onPressInternal}
-      testID={'PasteButton'}
-    />
+    <View style={styles.container}>
+      <Touchable onPress={onPressInternal} style={styles.button} testID={'PasteButton'}>
+        <>
```

Comment:
Is this fragment needed?

Reply:
seems to look and work fine without it, removed it